RecoParticleFlow/PFClusterProducer/plugins/PFMultiDepthClusterProducer.cc

```diff
@@ -67,6 +81,17 @@ PFMultiDepthClusterProducer::PFMultiDepthClusterProducer(const edm::ParameterSet
 }
 
 void PFMultiDepthClusterProducer::beginRun(const edm::Run& run, const edm::EventSetup& es) {
+  if (cutsFromDB) {
+    const HcalTopology& htopo = es.getData(htopoToken_);
+    const HcalPFCuts& hcalCuts = es.getData(hcalCutsToken_);
+
+    std::unique_ptr<HcalPFCuts> paramPF_;
+    paramPF_ = std::make_unique<HcalPFCuts>(hcalCuts);
+    paramPF_->setTopo(&htopo);
+    paramPF = paramPF_.release();
```

Comment:
Does this leak?

Reply:
I am not exactly sure how to check that.
I tried running with `SimpleMemoryCheck`, here is the log: 
https://swmukher.web.cern.ch/swmukher/mem_12434_0_TTbar_14TeV_2023.out
I am not fully sure how to interpret this log, but there is no mention of memory leak coming from PFMultiDepthClusterProducer.cc or PFClusterProducer.cc

Reply:
After ownership is released, who takes care of destroying the underlying object?

Reply:
SimpleMemoryCheck does not detect memory leaks.